src/components/PortForwarding.vue

```diff
@@ -3,6 +3,12 @@
   -->
 <template>
   <div>
+    <Checkbox
+      :label="'Include Kubernetes services'"
+      :value="includeKubernetesServices"
+      :disabled="notRunning"
```

Comment:
It might be better to do the negation here?
```suggestion
      :disabled="!isRunning"
```

Reply:
I understand why the attribute is called `disabled` and not `enabled` -- in the early days you would disable a tag in HTML 2.x/3.x with a developer-friendly `<input type=checkbox disabled label="click here">` -- no quotes needed for bareword values, no value needed for boolean attributes. But even since those days, I've found the logic to control disabling an element needs awkward negation, where an `enabled=BOOL` wouldn't. We've run into it here. I prefer keeping the Vue RHS expressions as simple as possible -- hence the `notRunning` method.

Reply:
Alright then, keeping it as `notRunning` it is :)